test/index.js

```diff
@@ -90,24 +96,12 @@ describe('DoubleClick Floodlight', function(){
   describe('track', function(){
```

Comment:
Yeah, just to make sure the early bailout works. I think that is good (with a `.end(done)`). It should assert that no requests were made.